rclpy/test/test_talker_listener.py

```diff
@@ -51,10 +52,10 @@ def test_rclpy_talker_listener():
     talker_process.start()
 
     import rclpy
-    from rclpy.qos import qos_profile_default
-
     rclpy.init([])
 
+    from rclpy.qos import qos_profile_default
```

Comment:
I haven't touched the this yet (this is part of @esteve's changes). I'll move this back into in progress until I refactor the tests.